hack/build/docker/builder/Dockerfile

```diff
@@ -1,14 +1,20 @@
-FROM quay.io/centos/centos:stream9
+FROM registry.fedoraproject.org/fedora-minimal:33
 MAINTAINER "The KubeVirt Project" <kubevirt-dev@googlegroups.com>
 
-RUN 	dnf -y install dnf-plugins-core && \
-	dnf config-manager --set-enable crb && dnf update -y && \
-	dnf install -y \
+COPY fedora.repo /tmp/fedora_ci.dnf.repo
+
+RUN sed -i 's/proxy = None//gI' /tmp/fedora_ci.dnf.repo && \
+	cat /tmp/fedora_ci.dnf.repo && \
+	mkdir /etc/yum.repos.d/old && \
+	mv /etc/yum.repos.d/*.repo /etc/yum.repos.d/old  && \
+	mv /tmp/fedora_ci.dnf.repo /etc/yum.repos.d/fedora.repo && \
```

Comment:
So 
```
sed -i 's/proxy = None//gI' /tmp/fedora_ci.dnf.repo && \
	cat /tmp/fedora_ci.dnf.repo && \
	mkdir /etc/yum.repos.d/old && \
	mv /etc/yum.repos.d/*.repo /etc/yum.repos.d/old  && \
	mv /tmp/fedora_ci.dnf.repo /etc/yum.repos.d/fedora.repo &&
```
Was needed a long time ago when we ran in the oVirt CI system. I don't think this is needed anymore.

Reply:
good point but I'd like to ignore for now.  Hopefully moving to centos stream soon after this